Source/Bricks/Collection/CollectionBrick.swift

```diff
@@ -51,6 +52,10 @@ public protocol CollectionBrickCellDataSource: class {
     func currentPageForCollectionBrickCell(_ cell: CollectionBrickCell) -> Int?
 }
 
+public protocol CollectionBrickCellDelegate: class {
```

Comment:
Why not make this a `BrickCellDelegate` (or `BrickDelegate`, based on my previous comment) that would allow developers to hook into this functionality for all bricks, not just those of type `CollectionBrick`?

Reply:
does my comments on slack make sense?

Reply:
Delegate and datasources are set on the brick